app/core/models.py

```diff
@@ -88,3 +89,18 @@ def set_permission_groups(self, group_list):
         else:  # viewer (least priv)
             reporter.user_set.remove(self)
             contributor.user_set.remove(self)
+
+
+class Notification(models.Model):
+    '''User notification model for alerting'''
```

Comment:
Need choices for notification type . Django 3 allows for TextChoices subclassing https://docs.djangoproject.com/en/3.1/ref/models/fields/#choices
See example with subclass YearInSchool. 
For now choice will just be 'email' but we will add more later.